File: src/frontend/src/app/models/room_booking.ts
// src/app/models/room_booking.ts
export class RoomBooking {
    constructor(
        public id: number,
        public startDate: string,
        public endDate: string,
        public cost: number,
        public discount: number,
        public fees: number,
        public status: string,
        public bookingId: string,
        public lodgingId: number,
        public roomNumber: number,
        public booking: any, // Cambia esto si tienes un tipo específico para Booking
        public room: any // Cambia esto si tienes un tipo específico para Room
    ) {}
}

File: src/backend/src/api/Models/RoomBooking.cs
namespace Restify.API.Models;

public class RoomBooking
{
	public DateOnly			StartDate { get; set; }
	public DateOnly			EndDate { get; set; }
	public decimal			Cost { get; set; }
	public decimal			Discount { get; set; }
	public decimal			Fees { get; set; }
	public BookingStatus	Status { get; set; }
	public int				BookingId { get; set; }
	public int				LodgingId { get; set; }
	public int				Id { get; set; }
	public uint				RoomNumber { get; set; }

	public Booking	Booking { get; }
	public Room		Room { get; }
}
